zerver/views/registration.py

```diff
@@ -333,6 +339,26 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
                                           source_profile=source_profile,
                                           realm_creation=realm_creation)
 
+        if realm_creation and settings.BILLING_ENABLED:
+            if settings.STAFF_SUBDOMAIN is not None:
+                try:
+                    staff_realm_uri = get_realm(settings.STAFF_SUBDOMAIN).uri
+                except Realm.DoesNotExist:
+                    return HttpResponseNotFound("Realm %s does not exist" % (settings.STAFFSUBDOMAIN,))
+            else:
+                return HttpResponse("STAFF_SUBDOMAIN not set correctly.")
+
+            realm_link = staff_realm_uri + reverse('analytics.views.support')
+            realm_link = realm_link + "/?q=" + user_profile.realm.string_id
+
+            send_email_to_support(realm_name=realm_name,
+                                  organization_type=organization_type,
+                                  repository_link=repository_link,
+                                  organization_nature=organization_nature,
+                                  organization_website=organization_website,
+                                  realm_link=realm_link)
```

Comment:
I think this whole block can be in the `send_email_to_support`, which perhaps should be renamed to `notify_support_new_realm`.